content/en/about-filecoin/managing-assets/index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Managing assets"
-description: "Many interactions with the Filecoin network require FIL to process. This page explains the different denominations of FIL, how to store it, and how to aquire it."
+title: "FIL token"
+description: "Many interactions with the Filecoin network require FIL to process. Additionally, participants in hackathons may receive FIL as payment for bounties and prizes. This page explains the different denominations of FIL, how to store it, and how to acquire it."
```

Comment:
```suggestion
description: "FIL is the native token of the Filecoin network. This page explains how to acquire, store, and use FIL tokens."
```
```suggestion
description: "Many interactions with the Filecoin network require FIL to process. Additionally, participants in hackathons may receive FIL as payment for bounties and prizes. This page explains the different denominations of FIL, how to store it, and how to acquire it."
```

Simplify a lot. Also, while hackathon payments is one use-case that's not well-served by the current docs, it should not be called out at the network documentation level.